tiny_dnn/nodes.h

```diff
@@ -342,15 +342,15 @@ class sequential : public nodes {
   friend class nodes;
 
   std::vector<tensor_t> normalize_out(
-    const std::vector<const tensor_t *> &out) {
+    const std::vector<const Tensor<> *> &out) {
     // normalize indexing back to [sample][layer][feature]
     std::vector<tensor_t> normalized_output;
 
     const size_t sample_count = out[0]->size();
     normalized_output.resize(sample_count, tensor_t(1));
 
     for (size_t sample = 0; sample < sample_count; ++sample) {
-      normalized_output[sample][0] = (*out[0])[sample];
+      normalized_output[sample][0] = ((*out[0])[sample]).toVec();
```

Comment:
I think ``toVec()`` has the extra computational cost of making a copy, is there any way to avoid this? Or this is just temporal as it is written in the ``toVec`` method?